ert_data/results.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+
+from ert_data import loader
+from ert_data.measured import MeasuredData, _add_index_range
+
+
+class Results:
+
+    def __init__(self, facade, keys, case):
+        self._facade = facade
+        self._case = case
+        self._set_data(self._get_data(keys))
```

Comment:
`self.data = self._get_data ...` ?

Reply:
Just an artifact of copy-pasta oriented programming :p trying to keep the code similar to the existing MeasuredData class. Not sure where we are going with this code to be honest, so not sure how to structure it. Perhaps we could just move the logic to the facade, and simply call it from the PlotApi. Let's discuss how it should be.